crates/nu-command/src/default_context.rs

```diff
@@ -179,8 +179,16 @@ pub fn add_shell_command_context(mut engine_state: EngineState) -> EngineState {
             Char,
             Decode,
             Encode,
+            DecodeBase64Old,
+            EncodeBase64Old,
+        DecodeBase32,
```

Comment:
Is there a reason this isn't aligned?
```suggestion
            DecodeBase32,
```

Reply:
Nope, my error.  I use tabs, and Cargo's formatter doesn't fix that inside macros.